README.md

```diff
@@ -3,6 +3,11 @@ SPHERE.IO - Snowflake
 
 [![Build Status](https://travis-ci.org/commercetools/sphere-snowflake.png)](https://travis-ci.org/commercetools/sphere-snowflake)
 
+:no_entry_sign: _**This shop template is deprecated!**_
+
+_This shop template is no longer maintained nor up to date, so please use [Sunrise](https://github.com/commercetools/sphere-sunrise) instead, the shop template for the [SPHERE JVM SDK](https://github.com/sphereio/sphere-jvm-sdk). If you are looking for a project using the older version [SPHERE Play SDK](https://github.com/commercetools/sphere-play-sdk), you can always check our [Fedora](https://github.com/commercetools/sphere-fedora) template._
```

Comment:
if there are advantages that sunrise has over snowflake, I would mention them similar as Hajo did for the JVM SDK. It is also fine if sunrise has no other advantage than being built on top of the new JVM SDK, but then make clear that it is the newer SDK and the only one being maintained from our side.